app.py

```diff
@@ -174,13 +208,28 @@ def prepare_metadata():
     sort_image_captions()
 
 
-# Deletes all files uploaded through the GUI and removes them from the dict
-def clean_up():
+# Deletes files uploaded through the GUI and removes them from the dict
```

Comment:
Why not use actual docstring format? https://www.python.org/dev/peps/pep-0257/

```
def clean_up_user_images(self):
    """Cleans user images.

    Deletes files uploaded through the GUI and removes them from the dict
    If 'self' is given then only the current user's images are deleted
    """
    img_prefix = get_user_img_prefix(self) if self else temp_img_prefix
```

Reply:
No experience with doc string, I'll switch to it, I assume it's a better standard